rust/hyperlane-core/src/traits/db.rs

```diff
@@ -34,7 +34,7 @@ where
     async fn retrieve_by_sequence(&self, sequence: u32) -> Result<Option<T>>;
```

Comment:
Nit: unrelated to this PR, but I think this `HyperlaneSequenceIndexerStore` trait should be renamed to `SequenceIndexerStoreReader`. Initially I thought the cursors had write access to the db due to this trait name, but turns out it's just reading for fast-forwarding purposes

Reply:
+1, I'll just rename, this is confusing